tests/test_parse.py

```diff
@@ -114,18 +113,22 @@ class MyModel(BaseModel):
     assert isinstance(m.__root__.__root__[0], str)
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_json():
-    assert Model.parse_raw('{"a": 12, "b": 8}') == Model(a=12, b=8)
+    # TODO: Are we planning to keep model_validate_json?
+    assert Model.model_validate_json('{"a": 12, "b": 8}') == Model(a=12, b=8)
 
 
 @pytest.mark.xfail(reason='working on V2')
 def test_json_ct():
+    # TODO: Should this test be dropped now that parse_raw is gone?
     assert Model.parse_raw('{"a": 12, "b": 8}', content_type='application/json') == Model(a=12, b=8)
 
 
 @pytest.mark.xfail(reason='working on V2')
 def test_pickle_ct():
+    # TODO: It seems model_validate_json has replaced parse_raw, except that it can _only_ be used with json
+    #   Are we just dropping the content_type/proto functionality? If so, should we drop these tests?
+    #   Do we need to document and/or create a suggested migration path?
     data = pickle.dumps(dict(a=12, b=8))
     assert Model.parse_raw(data, content_type='application/pickle', allow_pickle=True) == Model(a=12, b=8)
```

Comment:
Question 8: Seems we've decided to not support other kinds of content_type any more, should this test just be dropped now? Same question applies to the "proto" argument.

Reply:
I am horrified that we unpickled random data previously, I'm glad we're dropping it. I would vote for removing this test.

Reply:
well we had the required `allow_pickle` argument as a guard, but yes, I agree. :see_no_evil: 

Remove the test.